docs/time_series_for_support_staff.md

```diff
@@ -0,0 +1,159 @@
+Time Series For Devs/Sys-Admins
+===============================
+
+So Riak Time Series has gone live in your organisation? What now? What is it? How do you support it?
+
+You can view this document as a prezzo at:
+http://remarkjs.com/remarkise
+
+---
+
+Overview
+--------
+
+Time Series is a storage and query layer on top of Riak KV that provides new ways of managing data on the normal Riak platform.
+
+It also provides an SQL shell, `riak-shell`, which enables you to configure and query your Time Series system in an intuitive manner.
+
+At its core is a new type of bucket of provides a customisable degree of data co-location.
+
+---
+
+Riak KV Data Location
+---------------------
+
+In Riak KV you write data to a key and it goes and gets written at (usually) 3 vnodes on (usually) 3 physical nodes in your cluster. You write data to another key and that is written to a different set of 3 vnodes on a different set of 3 physical nodes.
+
+To query over a Key Value store like Riak KV then you need to scan the entire cluster.
+
+---
+
+Riak TS Data Colocation
+-----------------------
+
+With Riak TimeSeries you construct a composite key with a timestamp - and for a defined, definable duration (3 seconds, 15 minutes, 2 hours, 9 days) all data with the same composite root will be written to the same set of 3 vnodes on the same set of 3 physical nodes.
+
+Based on this data co-location you can then write queries that go to where ranges of data are stored and get back results.
+
+The querying of Time Series data is done via a quite limited, subset of SQL - so it works with your tools.
+
+---
+
+Differences With Riak KV
+------------------------
+
+Time Series is immutable data - once written it is never updated. So Riak TS uses the Write Once put path - this doesn't require a read-before-write and gives faster performance.
```

Comment:
"once written it is never updated" is the platonic ideal. Let's be more accurate here.